### PR TITLE
Refactor into separate package and extract constants

### PR DESCRIPTION
**Pull request by bergerdenes**
As requested.

### CODE REVIEW

File: template-manager-cmtemplate/src/test/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/hue/HueConfigProviderTest.java [hunk 1]
```diff
@@ -18,25 +18,26 @@
 import com.cloudera.api.swagger.model.ApiClusterTemplateConfig;
 import com.cloudera.api.swagger.model.ApiClusterTemplateVariable;
 import com.sequenceiq.cloudbreak.cmtemplate.CmTemplateProcessor;
+import com.sequenceiq.cloudbreak.cmtemplate.configproviders.HueConfigProvider;
```
Comment (adoroszlai) [inline template-manager-cmtemplate/src/test/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/hue/HueConfigProviderTest.java]: I guess `HueConfigProvider` should be moved to the subpackage, too.

Comment (bergerdenes) [reply template-manager-cmtemplate/src/test/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/hue/HueConfigProviderTest.java]: correct, I forget it (i was on the train...)

**Comment by lajosrodek** [comment]
LGTM 👍 .

**Comment by adoroszlai** [comment]
@bergerdenes can you please squash your commits?

(I prefer separate commits when addressing review comments, or to indicate logically separate sub-changes, as they make reviews easier.  But unfortunately this repo does not allow "Squash and merge" for PRs.)

**Comment by lajosrodek** [comment]
To be continued in #5009.

**Pull request closed without merging**